src/baseclient.js

```diff
@@ -283,6 +283,10 @@ BaseClient.prototype = {
    * Retrieve full URL of a document. Useful for example for sharing the public
    * URL of an item in the ``/public`` folder.
    *
+   * Please note: This method currently only works for remoteStorage
```

Comment:
reStructuredText has all kinds of directives for additional information, e.g. HINT, TIP, WARN, among others, which are then rendered nicely in the output. I'd go with a warning here.

Reply:
Thanks for the HINT :)